ios/RNTPTDocumentView.m

```diff
@@ -2090,5 +2090,25 @@ -(void)selectAnnotation:(NSString *)annotationId pageNumber:(NSInteger)pageNumbe
     return map;
 }
 
+#pragma mark - Set Current Page
+
+- (bool)setCurrentPage:(NSInteger)pageNumber {
+    
+    NSError *error = nil;
+    
+    __block bool setResult;
+    
+    [self.pdfViewCtrl DocLock:YES withBlock:^(PTPDFDoc * _Nullable doc) {
```

Comment:
Lock is not required as you are not modifying or even reading the document here.